src/lib/Tab/Dashboard/PagerContentToDataMapper.php

```diff
@@ -103,10 +104,14 @@ public function map(Pagerfanta $pager, ?string $forcedLanguage = null): array
                 'modified' => $content->versionInfo->modificationDate,
                 'initialLanguageCode' => $content->versionInfo->initialLanguageCode,
                 'content_is_user' => (new ContentIsUser($this->userService))->isSatisfiedBy($content),
-                'available_translations' => $this->languageService->loadLanguageListByCode(
-                    $availableLanguagesCodes
+                'available_translations' => $availableTranslations,
+                'available_enabled_translations' => array_filter(
+                    $availableTranslations,
+                    (function (\eZ\Publish\API\Repository\Values\Content\Language $translation) {
```

Comment:
`\eZ\Publish\API\Repository\Values\Content\Language` import is missing +

```suggestion
                    (static function (Language $translation): bool {
```